docs/tutorials/basics/part3.md

```diff
@@ -67,8 +67,14 @@ create table PRICE (
 }]');
 ```
 
-Start the pipeline.  The input connectors ingest data from S3 and push it to the pipeline.
-You should see the outputs appear in the Output tab.
+Select the `PREFERRED_VENDOR` view in the `Change Stream` tab and start the pipeline.
+The input connectors ingest data from S3 and push it to the pipeline.
+You should see the outputs appear in the `Change Stream` tab.
+
+:::note
+You must select the table or view you want to monitor in the `Change Stream` tab _before_ starting the pipeline
+in order to observe all updates to the view from the start of the execution.
```

Comment:
Leave a note that you can view the whole table (not the change stream) using the API at any time, and that this will be available in the UI in the coming weeks.